src/borg/testsuite/archiver/create_cmd.py

```diff
@@ -801,7 +801,7 @@ def extract_chunking_time(borg_create_output):
 
         # Test case set up: create a repository and a file
         self.cmd(f"--repo={self.repository_location}", "rcreate", RK_ENCRYPTION)
-        self.create_regular_file("testfile", contents=randbytes(5000000))
+        self.create_regular_file("testfile", contents=randbytes(5500000))
```

Comment:
How about just going to 10MB?

I guess increasing it just by 10% would still make it fail almost as frequently as it fails now.